metabrainz/templates/index/privacy.html

```diff
@@ -7,12 +7,144 @@ <h1 class="page-title">{{ _('Privacy Policy') }}</h1>
 
   <h2>{{ _('Summary') }}</h2>
   <p>
-    {{ _('The MetaBrainz Foundation will not collect any private information unless
-    you choose to donate to support the MusicBrainz project. Any personal
-    information you choose to provide during the donation process will not be
-    revealed to anyone else, unless you give us permission to list your name on
-    our donations page. We will not contact you unless you give us permission
-    to do so when you make a donation.') }}
+  <ol>
+    <li>
+      You do not have to provide any personal data to be able to browse the contents of the databases published by MetaBrainz.
+    </li>
+    <li>
+      You do not have to provide any personally-identifying information if you choose not to.
+    </li>
+    <li>
+      We will never reveal your email address on MetaBrainz controlled web sites.
+    </li>
+  </ol>
+  <p>
+    In order to get maximum value out of the MetaBrainz projects, you may want to create an account and log in. 
+    Doing so grants you the ability to: edit the databases, contribute your own data, communicate with other users, and keep track 
+    of recently released music from artists in your MusicBrainz collection. If you choose to create an account then the minimum we 
+    ask is that you: choose a unique username and password, use a web browser that accepts "session" cookies, and provide a verified 
+    email address.
```

Comment:
Technically, a verified email address isn't needed (for MusicBrainz), but not providing one just seriously hinders what you can do. Not sure if this distinction is significant enough to denote here though, so I'm happy to keep it the old way.

Reply:
Happy as is.